data/yam/agama/auto/template.jsonnet

```diff
@@ -0,0 +1,12 @@
+local lib = import 'lib/base.libsonnet';
+
+function(storage='default', product='SLES', scripts='default', encrypted=false) {
+  product: lib.getProduct(product),
```

Comment:
no need for a function here, let's use the external variable directly.

Reply:
Actually, it works like that, if you run `jsonnet --tla-str product="Tumbleweed" template.jsonnet` the output will contains Tumbleweed in the product id.
Example: 
```
> jsonnet --tla-str product="Tumbleweed" data/yam/agama/auto/template.jsonnet 
{
   "product": {
      "id": "Tumbleweed"
   },
   "root": {
      "hashedPassword": true,
      "password": "$6$vYbbuJ9WMriFxGHY$gQ7shLw9ZBsRcPgo6/8KmfDvQ/lCqxW8/WnMoLCoWGdHO6Touush1nhegYfdBbXRpsQuy/FTZZeg7gQL50IbA/"
   },
.
.
.
```

Reply:
The function allow us to set a default variable, avoid errors on empty of parameters run.

Reply:
Ok I removed the function, the code will be on template.